ideascube/mediacenter/templates/mediacenter/index.html

```diff
@@ -21,13 +21,43 @@
         </div>
     {% endif %}
     {% url 'mediacenter:index' as index %}
+    {% if q or kind or lang or tags %}
```

Comment:
All this will need a bit of CSS love ;)
I think the classic tag with coloured background and a 'x' to remove it will be ok.